packages/web_benchmarks/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.0.8
```

Comment:
Are there really no breaking changes here? That's very unusual for an NNBD transition, and I see public API that takes non-nullable values which looks breaking to me (since previously, e.g., `int` was equivalent to `int?` in NNBD code, not `int`).

Reply:
> Can you please direct me to it.

https://github.com/flutter/packages/pull/879/files#diff-97332f3a9c3e68b393026c3f408eb96ec266dd389c858623f87ad97096207a63R43-R45 for example, is changing three public parameters to non-nullable types.

> None of the changes look breaking to me.[recorder.dart#L66 ](https://github.com/amanv8060/packages/blob/d95ba8e16ab9af986f4c72c42f5b6558c2be0482/packages/web_benchmarks/lib/src/recorder.dart#L66) is one , but it was expected to be non null previously as well.

I haven't looked at all of the logic, which is why it's a question. Maybe in practice nothing changes, it's just unusual in an NNBD migration.

Reply:
I agree with @stuartmorgan. I think this needs to be version `0.1.0`. The dart docs recommend indicating the NNBD migration as breaking change:

> Update the version of the package to indicate a breaking change:
> 
> If your package is already at 1.0.0 or greater, increase the major version. For example, if the previous version is 2.3.2, the new version is 3.0.0.
>
> If your package hasn’t reached 1.0.0 yet, either increase the minor version or update the version to 1.0.0. For example, if the previous version is 0.3.2, the new version is either 0.4.0 or 1.0.0.

https://dart.dev/null-safety/migration-guide#package-version

Reply:
Note that there are (unsurprisingly) 0 published packages depending on this, which means the ecosystem cost of making this a breaking version change even if it doesn't turn out to be strictly necessary is essentially zero.

Reply:
@stuartmorgan since there are multiple such cases, i have added a generic line regarding breaking change. Also updated the version constraint